setup.py

```diff
@@ -0,0 +1,29 @@
+import sys
+from setuptools import setup
+
+if sys.version_info < (3, 6):
+    sys.exit('Sorry, tin-score-calculation requires Python >= 3.6')
+
+requirements = [
+    "bx-python>=0.8.10",
+    "guppy3>=3.1.0",
+    "matplotlib>=3.3.4",
+    "pandas>=1.2.3",
+    "psutil>=5.8.0",
+    "pysam>=0.16.0.1",
+    "RSeQC>=3.0.1",
+]
```

Comment:
Do we need to specify them here too? There is a `requirements.txt` file, it would be better to load this one here to avoid possible divergence. That's how I manage in conda env too (I also point to that txt).
Is that possible in a `setup.py`?

Reply:
`setup.py` is just a Python script, so maybe it could load the `requirements.txt` into a list?

Reply:
@AngryMaciek The issue is that the `requirements.txt` is very strict (`==` and not `>=`). If we can change the requirements.txt to `>=` then I could read in the requirements.txt and pass it as a variable in the setup.py. Would that be OK?

Reply:
Makes sense, I don't think `==` was critical for any reason...

Reply:
Ok, so the last commit fixes this.